README.md

```diff
@@ -229,16 +135,9 @@ Docker:
       --platform linux/amd64 \
       --build-arg VERSION=0.0.1 \
       --output type=docker \
-      --tag cisagov/example:0.0.1 .
+      --tag cisagov/admiral-docker:0.0.1 .
```

Comment:
Use correct image name.  Also, versions in the README should match the value in src/version.txt:
```suggestion
      --tag cisagov/admiral:1.3.0 .
```